### PR TITLE
feat: Display checkin app status information in the frontend to inform the user if there are problems with a connected data provider

### PR DESCRIPTION
**Pull request by mad-nuts**
Refs iris-connect/iris-backlog#221

### CODE REVIEW

File: iris-client-bff/src/main/java/iris/client_bff/config/RPCClientConfig.java [hunk 1]
```diff
@@ -18,6 +18,7 @@
 import javax.net.ssl.SSLContext;
 import javax.net.ssl.TrustManager;
 import javax.net.ssl.X509TrustManager;
+import javax.validation.constraints.NotNull;
```
Comment (jekutzsche) [inline iris-client-bff/src/main/java/iris/client_bff/config/RPCClientConfig.java]: The Constraints from `javax.validation` have only effect with`@Validated` at the class. I think here the annotation of Lombok is sufficient, as it is used for the other fields.

**Review by lucky-lusa — DISMISSED** [review]
LGTM

**Review by jekutzsche — APPROVED** [review]

**Pull request merged**